server/game/gamesteps/AttackFlow.js

```diff
@@ -23,6 +23,9 @@ class AttackFlow extends BaseStepWithPipeline {
             new SimpleStep(this.game, () =>
                 this.game.raiseEvent('onDefendersDeclared', { attack: this.attack })
             ),
+            new SimpleStep(this.game, () =>
```

Comment:
the next step was not an event to we needed this new step (or the ability to distinguish between WHEN and AFTER for an event, which doesnt seem to exist)

Reply:
when vs after are usually determined by using interrupt (when) vs reaction ( after).
so flashstrike would be the reaction, and sunshield would be the interrupt.

Can you see if that works?

Reply:
that worked! i didn't realize